roles/public_facing/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+# ansible_host changes to 'localhost' in the next task.  Set target_host
```

Comment:
oh, because local_action means 'it's running on localhost', but you need to have the name of the ultimate target host? If so, just change this comment to mention "local_action in the next task makes ansible_host be the local hostname" (or 'localhost' if that was a literal statement)